gluten-core/src/main/scala/io/glutenproject/backendsapi/Backend.scala

```diff
@@ -38,4 +38,6 @@ trait Backend {
   def broadcastApi(): BroadcastApi
 
   def settings(): BackendSettingsApi
+
+  def testApi(): TestApi
```

Comment:
Hi @marin-ma, as this interface is just for test use, can we move it to some place in test module? Maybe, into `BackendTestSettings`? Thanks!